tmp/README.txt

```diff
@@ -0,0 +1,21 @@
+Some resources for experimenting with strimzi-canary
+
+1. Preparation
+Make sure that you have your strimzi-canary docker image. If not, build strimzi-canary and
+make its image available to your cluster.
```

Comment:
not sure we really need this pre-requisite. User can use the official canary image, not building a custom one, or?